components/Account/Access/Campaigns/Grant.js

```diff
@@ -0,0 +1,25 @@
+import React, { Component } from 'react'
+
+import { Item } from '../../../List'
+
+import { timeFormat } from '../../../../lib/utils/format'
+
+import Revoke from './Revoke'
+
+const dayFormat = timeFormat('%e. %B %Y')
+
+class Grants extends Component {
+  render () {
+    const { grant, revokeAccess } = this.props
+
+    return (
+      <Item key={grant.id}>
+        an <strong>{grant.email}</strong><br />
```

Comment:
Avoid native `strong`. This leads to [faux bold](https://alistapart.com/article/say-no-to-faux-bold). Use [`Interaction.Emphasis`](https://styleguide.republik.ch/typography#emphasis-1)